app/gui2/src/components/ComponentBrowser.vue

```diff
@@ -344,8 +344,9 @@ function applySuggestion(component: Opt<Component> = null): SuggestionEntry | nu
   const providedSuggestion =
     component != null ? suggestionDbStore.entries.get(component.suggestionId) : null
   const suggestion = providedSuggestion ?? selectedSuggestion.value
-  if (suggestion == null) return null
-  input.applySuggestion(suggestion)
+  const suggestionId = component?.suggestionId ?? selectedSuggestionId.value
+  if (suggestion == null || suggestionId == null) return null
+  input.applySuggestion(suggestionId)
```

Comment:
Is here still a need for obtaining `suggestion`? We could just pass suggestionId and `applySuggestion` should handle that.

Reply:
Done